commands/outliers.js

```diff
@@ -0,0 +1,79 @@
+'use strict'
+
+const co = require('co')
+const cli = require('heroku-cli-util')
+const psql = require('../lib/psql')
+
+function * ensurePGStatStatement (db) {
+  let query = `
+SELECT exists(
+  SELECT 1 FROM pg_extension e LEFT JOIN pg_namespace n ON n.oid = e.extnamespace
+  WHERE e.extname='pg_stat_statements' AND n.nspname = 'public'
+) AS available`
+  let output = yield psql.exec(db, query)
+
+  if (!output.includes('t')) {
+    throw new Error(`pg_stat_statements extension need to be installed in the public schema first.
+This extension is only available on Postgres versions 9.2 or greater. You can install it by running:
```

Comment:
Fortunately, we only run Postgres version 9.2 or greater so you can drop part of this error message.

Reply:
Maybe we can say something like this instead: "This extension may not be available on hobby tier Postgres."
In fact, it's not "may not be available", but I guess "is not available"?
https://trello.com/c/BK0cdQH3/2045-pg-extras-that-need-it-should-check-for-pgstatstatements-and-create-the-extension

Reply:
It is 'available'. All postgres that we run, including yobukos have it as a 'shared_preload_libraries'. 

Why would we not install it automatically here?

Reply:
I'm not opposed to automatically install, but I also want to warn it may make customers confusing as it'll return "zero" result and they don't know why (well we can say why explicitly I guess).
"available" part, you're correct, I meant "may not be installed" or "is not installed".

Reply:
>This view, and the function pg_stat_statements_reset, are available only in databases they have been specifically installed into by installing the pg_stat_statements extension. However, statistics are tracked across all databases of the server whenever the pg_stat_statements module is loaded into the server, regardless of presence of the view.

So we load the library in shared_preload_libraries and install the extension in the main db. As I read the docs, it should be tracking queries and gathering statistics no matter if the extension is installed in the hobby database. A 'zero' result, then, should be true.

Reply:
oh interesting. yeah maybe worth double checking if it's correct or not with existing hobby databases. I'm gonna be PTO from tomorrow so I'll let you guys decide what to do. As long as it is not confusing to people and works, I'm fine with options that you suggested including auto-install.

Reply:
For now, let's do the simple thing and strike `This extension is only available on Postgres versions 9.2 or greater. ` from the error message. Auto-installing is a separate issue; let's handle it in a separate patch.

Reply:
After thinking about this some more, I'm okay with doing auto-install in this patch, too. Either works.